### DISCUSSION TITLE
(question) Possible to use AnimatedPoint for MarkerView / PointAnnotation?

### DISCUSSION THREAD

**Discussion by mtnptrsn** [rnmapbox/maps#1329] [Q&A]
I can see there is class called AnimatedPoint, but it is not documented. Is it somehow possible to animate the position of a MarkerView or a PointAnnotation?

**Comment by AndreasAtakan** [comment]
I am trying to do something similar.
Any feedback?

@mfazekas

**Reply by mfazekas** [reply]
I don't see why it would not work, but make sure you use an RNAnimated.createAnimatedComponent(MarkerView)
